src/dvclive/live.py

```diff
@@ -370,14 +371,25 @@ def log_metric(
         logger.debug(f"Logged {name}: {val}")
 
     def log_image(self, name: str, val):
-        if not Image.could_log(val):
-            raise InvalidDataTypeError(name, type(val))
-
+        # If we're given a path, try loading the image first. This might error out.
         if isinstance(val, (str, Path)):
             from PIL import Image as ImagePIL
 
             val = ImagePIL.open(val)
 
+        # If the provided image name does not have a format on it,
+        # try to infer the format from PIL Image.
+        if (
+            len(name.split(".")) <= 1
+            and isinstance_without_import(val, "PIL.Image", "Image")
+            and f".{str(val.format).lower()}" in Image.suffixes
+        ):
+            name = f"{name}.{str(val.format).lower()}"
```

Comment:
As mentioned above, I think we could drop this since I don't think it works for anything but existing image paths.